mlkem/verify.h

```diff
@@ -9,66 +9,227 @@
 #include "cbmc.h"
 #include "params.h"
 
-#define verify MLKEM_NAMESPACE(verify)
+#if !defined(CBMC) && !defined(_WIN32)
```

Comment:
Add a comment here in which case which approach is being used?

Reply:
Yes

Reply:
Thanks for that ellaborate comment - that's really useful!